entries/entry03.md

```diff
@@ -1,8 +1,52 @@
 # Entry 3
-##### X/X/XX
+##### 1/5/2020
+
+From the last time I have done a blog entry, not much has been done besides doing more research and learning more basics of another programming language.
+
+This time I began to learn **cocoapods**.
+
+### Cocoapod
+#### Why is it important for me specifically to learn this?
+
+While trying to figure out how to properly install firebase into firebase the official firebase website gave tips and directions on how to instal/implement firebase.
+In the [official website](https://firebase.google.com/docs/ios/setup?authuser=0) it is encouraged to use cocoapods in order to install firebase libraries into your program.
+
+**For what reason?**
+
+Cocoapods make things more efficient and organized when trying to properly handle the firebase.
+
+**What did I try learning about cocoapods?**
+
+For one: The basics.
+1. How to install it
+2. Familiarize the code
+3. Knowing where to go if I get stuck
+
+**1: Installing Cocoapods**
+
+While firebase was helpful in providing easy to read instructions on how to install cocoapods. I found it more helpful to go to the [official cocoapods website](https://guides.cocoapods.org/using/getting-started.html).
+
+The official website gave more detailed instructions in my opinion. The website also gave information about other possible useful items. An example of this being that the website reference ruby and how it is used along with cocoapods.
+
+While I myself do not know anything about ruby. It will come in handy when familiarizing the code for cocoapods.
+
+**2: Familiarizing the code**
+
+The syntax for cocoapods is not that hard to grasp. It is similar to that of git in IDE's. Although because it is still a new programming language that I have exposed myself to I do still need a few more days to completely soak it in.
```

Comment:
Show some code snippets! Let us see the code you're trying. This will be strong evidence of learning!